wagtail_localize_rws_languagecloud/rws_client.py

```diff
@@ -8,6 +8,20 @@
 from django.conf import settings
 
 
+def rws_language_code(language_code):
+    """
+    Returns the mapped RWS language code, if found in the LANGUAGE_CODE_MAP setting.
+    Defaults to the original value.
+    """
+    language_code_map = settings.WAGTAILLOCALIZE_RWS_LANGUAGECLOUD.get(
+        "LANGUAGE_CODE_MAP", {}
+    )
+    try:
+        return language_code_map.get(language_code, language_code)
+    except AttributeError:
```

Comment:
Is this try-except really needed? I am pretty sure the `get` method of a dict does not throw an AttributeError in any case, since if the key does not exist the default value will be returned

Reply:
I forgot we had system checks for the settings. Removed this and added a system check to ensure the map setting is correctly set up. Plus added a test for the system checks